client/my-sites/hosting/php-version-card/index.js

```diff
@@ -0,0 +1,226 @@
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import Spinner from 'components/spinner';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import { errorNotice, successNotice } from 'state/notices/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const requestId = ( siteId, method ) => `hosting-php-version-${ method }-${ siteId }`;
+
+export const requestPhpVersion = siteId => {
+	const method = 'GET';
+
+	return requestHttpData(
+		requestId( siteId, method ),
+		http(
+			{
+				method,
+				path: `/sites/${ siteId }/hosting/php-version`,
+				apiNamespace: 'wpcom/v2',
+				body: {},
+			},
+			{}
+		),
+		{
+			fromApi: () => version => {
+				return [ [ requestId( siteId, method ), version ] ];
+			},
+			freshness: 0,
+		}
+	);
+};
+
+export const setPhpVersion = ( siteId, version ) => {
+	const method = 'POST';
+
+	return requestHttpData(
+		requestId( siteId, method ),
+		http(
+			{
+				method,
+				path: `/sites/${ siteId }/hosting/php-version`,
+				apiNamespace: 'wpcom/v2',
+				body: {
+					version: version,
+				},
+			},
+			{}
+		),
+		{
+			fromApi: () => success => {
+				if ( true === success ) {
+					requestPhpVersion( siteId );
+				}
+				return [ [ requestId( siteId, method ), success ] ];
+			},
+			freshness: 0,
+		}
+	);
+};
+
+const PhpVersionCard = ( {
+	translate,
+	siteId,
+	loading,
+	updating,
+	updateResult,
+	version,
+	showErrorNotice,
+	showSuccessNotice,
+} ) => {
+	const [ selectedPhpVersion, setSelectedPhpVersion ] = useState( null );
+
+	useEffect( () => {
+		requestPhpVersion( siteId );
+	}, [] );
+
+	useEffect( () => {
+		const updateNoticeId = 'hosting-php-version';
+
+		if ( updateResult === 'failure' ) {
+			showErrorNotice( translate( 'Failed to set PHP version.' ), {
+				id: updateNoticeId,
+			} );
+		}
+
+		if ( updateResult === 'success' ) {
+			showSuccessNotice(
+				translate( 'PHP version successfully set to %(version)s.', {
+					args: {
+						version: selectedPhpVersion,
+					},
+				} ),
+				{
+					id: updateNoticeId,
+					showDismiss: false,
+					duration: 5000,
+				}
+			);
+		}
+	}, [ updateResult ] );
+
+	const changePhpVersion = event => {
+		const newVersion = event.target.value;
+
+		setSelectedPhpVersion( newVersion );
+	};
+
+	const getPhpVersions = () => {
+		return [
+			{
+				label: translate( '7.2', {
+					context: 'PHP Version',
+				} ),
+				value: '7.2',
+			},
+			{
+				label: translate( '7.3 (recommended)', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.